### DISCUSSION TITLE
본문 안 제목 굵게 처리 강제를 해제하고 싶습니다

### DISCUSSION THREAD

**Discussion by mmcrs** [pronist/hELLO#201] [Q&A] [answered]
![ddddd](https://user-images.githubusercontent.com/85152618/220826107-b8ff0cbe-da7c-41dc-aa6c-97aa27039977.png)

[해당 글](https://mmcrs.in/508) 중 하나를 보았을 때 빨간색 밑줄 쳐진 부분은 굵게 처리를 하지 않은 부분인데, 작성하고 보니 모두 굵게 처리 되었습니다.
에디터에서 작성한 그대로 서식 유지를 하고싶은데 어느 부분의 코드를 수정해야하나요?

**Comment by pronist** [comment, marked as answer]
@mmcrs 안녕하세요.

본문 제목에 대한 처리는 원본 소스에서 아래의 코드에 해당합니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/17f2d7d5e0e126d532087e36041585429f12272f/views/Main/Post/Permalink/Post/Content.pug#L306-L314
